stream.go

```diff
@@ -107,13 +75,35 @@ type ClientStream interface {
 	// CloseSend closes the send direction of the stream. It closes the stream
 	// when non-nil error is met.
 	CloseSend() error
-	// Stream.SendMsg() may return a non-nil error when something wrong happens sending
-	// the request. The returned error indicates the status of this sending, not the final
-	// status of the RPC.
+	// Context returns the context for this stream.
+	Context() context.Context
+	// SendMsg is generally called by generated code. On error, SendMsg aborts
+	// the stream and returns an io.EOF; the status may be determined by
+	// inspecting the error returned by calling RecvMsg.
+	//
+	// SendMsg blocks until:
+	//   - It schedules m with the transport, or
```

Comment:
How about:  "- There is sufficient flow control to schedule m with the transport, or"

...or something to indicate that we aren't just letting people queue up a billion messages.

Reply:
Done.